go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/hashicorp/vault/api v1.0.5-0.20200902155336-f9d5ce5a171a
 	github.com/kubernetes-csi/csi-lib-utils v0.9.1
-	github.com/kubernetes-csi/external-snapshotter/v2 v2.1.1
+	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.1.0
```

Comment:
as we have 4.1.1 https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v4.1.1 can  we update this to 4.1.1?

Reply:
@Madhu-1 eventhough the tag and release is available, the client package resolution to `4.1.1` was not happening to me. Is it resolving/importing `4.1.1` properly at your end ?

Reply:
what is the error. Maybe we can check with the kube-csi team?

Reply:
here it is : ```go: github.com/kubernetes-csi/external-snapshotter/client/v4@v4.1.1: reading github.com/kubernetes-csi/external-snapshotter/client/client/go.mod at revision client/v4.1.1: unknown revision client/v4.1.1``` , I can check with csi team though

Reply:
I agree that it is a little strange, and ideally we should use v4.1.1.

That said there is only kubernetes-csi/external-snapshotter#515 addressed, which does not seem to affect the client package.